keras/utils/data_utils.py

```diff
@@ -327,7 +326,7 @@ def __init__(self, x_set, y_set, batch_size):
                 self.batch_size = batch_size
 
             def __len__(self):
-                return math.ceil(len(self.x) / self.batch_size)
+                return (len(self.x) + self.batch_size - 1) // self.batch_size # round up
```

Comment:
New version is harder to read, which is a problem for a docstring code example, meant to be as immediately clear as possible. Maybe just use `np.ceil` instead and remove the import?